src/dfmt/ast_info.d

```diff
@@ -448,6 +452,51 @@ final class FormatVisitor : ASTVisitor
         ternaryExpression.accept(this);
     }
 
+    override void visit(const FunctionCallExpression functionCall)
+    {
+        // Check if function has any arguments.
+        if (functionCall.arguments.namedArgumentList is null)
+        {
+            functionCall.accept(this);
+            return;
+        }
+
+        /+
+        Items are function arguments: f(<item>, <item>);
+        Iterate them and check if they are named arguments: tok!":" belongs to a
+        named argument if it is preceeded by one tok!"identifier" (+ any number
+        of comments):
+        +/
+        foreach (item; functionCall.arguments.namedArgumentList.items)
+        {
+            // Set to true after first tok!"identifier".
+            auto foundIdentifier = false;
+
+            foreach (t; item.tokens)
+            {
+                if (t.type == tok!"comment")
```

Comment:
comment tokens are not part of the `tokens` array in the AST, they are only used in trivia tokens (leadingTriviaTokens, trailingTriviaTokens) and are attached to the regular tokens, so you don't have to check for them

Reply:
Thanks for the info, I will remove the comment check.